src/Symfony/Component/Dotenv/Dotenv.php

```diff
@@ -45,10 +45,11 @@
      * @throws FormatException when a file has a syntax error
      * @throws PathException   when a file does not exist or is not readable
      */
-    public function load($path/*, ...$paths*/)
+    public function load($path, ...$paths)
```

Comment:
Indeed, can sounds like calling `load()` will automagically look for some `.env` but nothing happens.